aiosqlite/core.py

```diff
@@ -283,6 +283,9 @@ def total_changes(self) -> int:
     async def load_extension(self, path: str):
         await self._execute(self._conn.load_extension, path)  # type: ignore
 
+    async def set_progress_handler(self, handler: Callable, n: int) -> None:
```

Comment:
Looking at the sqlite3 docs, this would be better annotated as `handler: Callable[[], Optional[int]]`.

Reply:
Done